hedera-node/test-clients/src/main/resource/contract/contracts/NumericContract/NumericContract.sol

```diff
@@ -147,4 +147,41 @@ contract NumericContract {
         }
     }
 
+    /*´:°•.°+.*•´.*:˚.°*.˚•´.°:°•.°•.*•´.*:˚.°*.˚•´.°:°•.°+.*•´.*:*/
+    /*                    HАS functions                    */
+    /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
+    function hbarApproveProxy(address spender, int256 amount) external {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("hbarApprove(address,int256)", spender, amount));
+
+        if (success == false) {
+            revert();
+        }
+    }
+
+    function hbarApprove(address owner, address spender, int256 amount) external {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("hbarApprove(address,address,int256)", owner, spender, amount));
+
+        if (success == false) {
+            revert();
+        }
+    }
+
+    /*´:°•.°+.*•´.*:˚.°*.˚•´.°:°•.°•.*•´.*:˚.°*.˚•´.°:°•.°+.*•´.*:*/
+    /*                    Exchange Rate functions                    */
+    /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
+    function convertTinycentsToTinybars(uint256 tinycents) external {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("convertTinycentsToTinybars(uint256)", tinycents));
+
+        if (success == false) {
+            revert();
+        }
+    }
+
+    function convertTinybarsToTinycents(uint256 tinybars) external {
+        (bool success, bytes memory result) = address(0x167).call(abi.encodeWithSignature("convertTinybarsToTinycents(uint256)", tinybars));
+
+        if (success == false) {
+            revert();
+        }
+    }
 }
```

Comment:
File needs trailing `\n`.  (You can configure your IDE to do this ...)

Reply:
It usually adds it. I guess it might be related to the merge conflicts.